### PR TITLE
feat: [CHAOS-2085]: Add support for Chaos 1.15.x

### PR DESCRIPTION
**Pull request by sagarkrsd**
feat: [CHAOS-2085]: Add support for Chaos 1.15.x

[CHAOS-2085]: https://harness.atlassian.net/browse/CHAOS-2085?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/chaos-common/templates/config.yaml [hunk 1]
```diff
@@ -10,10 +10,17 @@ data:
...
   NG_MANAGER_ENDPOINT: 'http://ng-manager:7090'
   CHAOS_MANAGER_ENDPOINT: 'http://chaos-manager-service:8080'
   ACCESS_CONTROL_ENDPOINT: 'http://access-control:9006/api'
+  REDIS_EVENTS_FWK_NAMESPACE: ''
+  REDIS_LOCK_ENDPOINT: 'redis://redis-sentinel-harness.{{ .Release.Namespace }}:6379'
+  REDIS_LOCK_SSL_ENABLED: 'false'
+  REDIS_LOCK_NAMESPACE: ''
+  REDIS_LOCK_DATABASE: '0'
+  NG_MANAGER_SVC_ENDPOINT: 'http://ng-manager:7090'
+  REDIS_STREAM_ENV: 'smp'
```
Comment (sagarkrsd) [inline src/chaos-common/templates/config.yaml:26]: validate this value...

File: src/chaos-manager/templates/job.yaml [hunk 2]
```diff
@@ -26,9 +26,9 @@ spec:
               name: chaos-db-upgrade-job-secret
         env:
           - name: PREV_VERSION
-            value: "0.12.0"
```
Comment (sagarkrsd) [inline src/chaos-manager/templates/job.yaml:29]: need to check for content of secret `db-upgrade-job-secret`

Comment (sagarkrsd) [reply src/chaos-manager/templates/job.yaml:29]: done

**Review by Adarshkumar14 — APPROVED** [review]

File: src/chaos-common/templates/config.yaml [hunk 3]
```diff
@@ -10,10 +10,18 @@ data:
...
   LOG_SERVICE_EXTERNAL_ENDPOINT: '{{ .Values.global.loadbalancerURL }}/gateway/log-service'
   NG_MANAGER_ENDPOINT: 'http://ng-manager:7090'
   CHAOS_MANAGER_ENDPOINT: 'http://chaos-manager-service:8080'
   ACCESS_CONTROL_ENDPOINT: 'http://access-control:9006/api'
+  REDIS_EVENTS_FWK_NAMESPACE: ''
+  REDIS_LOCK_ENDPOINT: 'redis://redis-sentinel-harness.{{ .Release.Namespace }}:6379'
+  REDIS_LOCK_SSL_ENABLED: 'false'
+  REDIS_LOCK_NAMESPACE: ''
+  REDIS_LOCK_DATABASE: '0'
+  NG_MANAGER_SVC_ENDPOINT: 'http://ng-manager:7090'
```
Comment (Jonsy13) [inline src/chaos-common/templates/config.yaml:25]: Reminder - We can test and remove based on testing in next release.

File: src/chaos-manager/values.yaml [hunk 4]
```diff
@@ -95,44 +95,46 @@ litmusChaosOperatorImage:
...
 chaosGamedayLimit: '200'
 
 chaosHubSize: '200'
 
-parallelChaosExperimentRuns: '25'
+parallelChaosExperimentRuns: '500'
 
 skipSecureVerify: 'true'
 
+expHelperImageVersion: "0.13.0"
```
Comment (Jonsy13) [inline src/chaos-manager/values.yaml:136]: 1.15.0

**Review by Jonsy13 — APPROVED** [review]

**Review by Adarshkumar14 — APPROVED** [review]

**Pull request merged**